config/initializers/httparty.rb

```diff
@@ -1,6 +1,6 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
-require 'httparty'
-require 'httparty/logger/mailchimp_logger'
-require 'httparty/logger/full_contact_logger'
-::HTTParty::Logger.add_formatter(:mailchimp, HTTParty::Logger::MailchimpLogger)
-::HTTParty::Logger.add_formatter(:full_contact, HTTParty::Logger::FullContactLogger)
+
+Rails.application.config.after_initialize do
+  ::HTTParty::Logger.add_formatter(:mailchimp, ::HTTParty::Logger::MailchimpLogger)
```

Comment:
@caseyhelbling I corrected the inflections so everything is `HTTParty` now instead of some being `Httparty`. The problem was due to the fact that initializers can't directly refer to autoloadable constants in initializers as explained [here](https://guides.rubyonrails.org/autoloading_and_reloading_constants.html#autoloading-when-the-application-boots). This was possible to work around the way I did originally but broke in Rails 7. Once I realized this, I backported the change here.